model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -647,29 +647,26 @@ message StandardCoders {
 
 // Experimental: A representation of a Beam Schema.
 message Schema {
-  enum TypeName {
+  enum AtomicType {
```

Comment:
Isn't 0 in an enum supposed to be reserved for unknown? It is wise, because of defaulting in proto libs.

Reply:
Yes, to my knowledge specifically adding an UNSPECIFIED with a value of 0 will make this clearer.
For example: https://github.com/apache/beam/blob/eb5a8c2cbd973972491dedef23aff98e450a9f5a/model/pipeline/src/main/proto/beam_runner_api.proto#L451

Reply:
Done.